src/Raven.Server/Web/Authentication/AdminCertificatesHandler.cs

```diff
@@ -489,23 +490,44 @@ public async Task GetCertificates()
                     {
                         const string serverCertificateName = "Server Certificate";
                         // The server cert is not part of the local state or the cluster certificates, we add it to the list separately
-                        if ((name == null || name == serverCertificateName) && Server.Certificate.Certificate != null)
+                        if ((name == null || name == serverCertificateName) && Server.Certificate.ServerCertificate != null)
                         {
                             var serverCertDef = new CertificateDefinition
                             {
                                 Name = serverCertificateName,
-                                Certificate = Convert.ToBase64String(Server.Certificate.Certificate.Export(X509ContentType.Cert)),
+                                Certificate = Convert.ToBase64String(Server.Certificate.ServerCertificate.Export(X509ContentType.Cert)),
                                 Permissions = new Dictionary<string, DatabaseAccess>(),
                                 SecurityClearance = SecurityClearance.ClusterNode,
-                                Thumbprint = Server.Certificate.Certificate.Thumbprint,
-                                PublicKeyPinningHash = Server.Certificate.Certificate.GetPublicKeyPinningHash(),
-                                NotAfter = Server.Certificate.Certificate.NotAfter,
-                                NotBefore = Server.Certificate.Certificate.NotBefore
+                                Thumbprint = Server.Certificate.ServerCertificate.Thumbprint,
+                                PublicKeyPinningHash = Server.Certificate.ServerCertificate.GetPublicKeyPinningHash(),
+                                NotAfter = Server.Certificate.ServerCertificate.NotAfter,
+                                NotBefore = Server.Certificate.ServerCertificate.NotBefore
                             };
 
                             var serverCert = context.ReadObject(serverCertDef.ToJson(metadataOnly), "Server/Certificate/Definition");
 
-                            certificateList.TryAdd(Server.Certificate.Certificate?.Thumbprint, serverCert);
+                            certificateList.TryAdd(Server.Certificate.ServerCertificate?.Thumbprint, serverCert);
+
+                            var certificateForCommunication = Server.Certificate.ClientCertificate;
+                            if (certificateForCommunication.Thumbprint != Server.Certificate.ServerCertificate?.Thumbprint)
+                            {
+                                var serverCertForCommunicationDef = new CertificateDefinition
+                                {
+                                    Name = "Server Certificate for communication " + ServerStore.NodeTag,
```

Comment:
Wondering if this is how we want to display it to the user. Thinking about something like "Node A (internal)". Thoughts?

Reply:
talked and decided to leave it as is